Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -811,7 +811,13 @@ void RenderLayer::rebuildZOrderLists(std::unique_ptr<Vector<RenderLayer*>>& posZ
             if (!posZOrderList)
                 posZOrderList = makeUnique<Vector<RenderLayer*>>();
 
+            RenderLayer* viewTransitionLayer = nullptr;
```

Comment:
```suggestion
            RenderLayer* viewTransitionLayer;
```

Reply:
A stack pointer won't be default initialised, I think we need this.